src/apps/relay/mainrelay.c

```diff
@@ -1653,6 +1653,11 @@ static void read_config_file(int argc, char **argv, int pass)
 		} else
 			TURN_LOG_FUNC(TURN_LOG_LEVEL_WARNING, "WARNING: Cannot find config file: %s. Default and command-line settings will be used.\n",
 				config_file);
+
+		if (full_path_to_config_file) {
+			free(full_path_to_config_file);
```

Comment:
I think it would be more code style conform to use "turn_free" instead of simple "free"

Reply:
That's fixed. Thanks